.github/workflows/publish-python-client.yml

```diff
@@ -166,13 +164,11 @@ jobs:
       - name: Publish osparc to Test PyPI
```

Comment:
Do i see it correctly that pushing to main pypi will happen even in not in master (but just a tagged commit?).
Or did I miss some if requirement somewhere?
Should we maybe push to test pypi inside PRs, and main pypi in master?

Reply:
Right, triggering the publishing will happen by tagging the commit. How does that sound to you? I had a discussion with Pedro about the publishing to TestPyPi. The reason why I didn't do that for every commit is that (as far as I understand) one has to come up with a naming convention for the different comments (so I guess the hash), but that seemed a bit cumbersome to me. What I wanted to do instead was use the Github packaging system, but they currently don't support Python packaged. The last alternative (which is what I currently use) is to have a script which fetches the latest master build of the client directly from the artifacts generated on the CI. For that I use github cli. I think that is the most convenient way, but I am not so familiar with testpypi so we could pursue that if you feel it is worth it.

Reply:
Yeah, sure, that's also fine. Should we maybe require being in master 'and' tagging the commit? Just so that there is no chance a pypi push gets triggered from within a PR>